document/src/main/java/com/ritense/document/web/rest/impl/JsonSchemaDocumentDefinitionResource.java

```diff
@@ -61,6 +61,22 @@ public ResponseEntity<Page<? extends DocumentDefinition>> getDocumentDefinitions
         return ok(documentDefinitionService.findAll(fixPageable(pageable)));
     }
 
+    @Override
+    public ResponseEntity<String> getDocumentDefinitionTemplate() {
```

Comment:
I guess the story was not clear. The `$id` and `title` should be provided by the FE and replaced in the template. I believe that also means this should be a POST?

Reply:
Fixed